rudder-lang/src/parser/tests.rs

```diff
@@ -259,14 +264,36 @@ fn test_penum() {
             }
         ))
     );
+    assert_eq!(
+        // to test preceeding megtadata, directly test a pdeclaration
+        map_res(penum, "enum abc3 { @metadata=\"value\"\n a, b, }"),
```

Comment:
metaada should only be parsed at the begining of a line

Reply:
fixed :)